### PR TITLE
Suggested bug fixes

### LINKED ISSUE #1: Class 'postParser' not found
I get a 500 error if I access 'memberlist.php' and 'showteam.php'

PHP Fatal error:  Class 'postParser' not found in /home/xxxxx/public_html/xxxxx/inc/plugins/asb/modules/poll_box.php on line 156

I deleted those 2 scripts from the ACP (adv sidebox manage scripts) and now I do not get any error.

### PR DESCRIPTION
**Pull request by WildcardSearch**
#1 where library failed to be included
#2 where templates are given an sid of -1 but recent updates have

changed that

### CODE REVIEW

File: Upload/inc/plugins/asb/modules/poll_box.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ function asb_poll_box_info()
 		"title" => $lang->asb_poll_box_title,
 		"description" => $lang->asb_poll_box_desc,
 		"wrap_content"	=> true,
-		"version" => "1",
+		"version" => "1.0.2",
```
Comment (WildcardSearch) [inline Upload/inc/plugins/asb/modules/poll_box.php:36]: I've incremented your version # to help with module upgrade

File: Upload/inc/plugins/asb/modules/poll_box.php [hunk 2]
```diff
@@ -88,8 +88,6 @@ function asb_poll_box_info()
 					</tr>
 				</form>
 EOF
-				,
-				"sid" => -1
```
Comment (WildcardSearch) [inline Upload/inc/plugins/asb/modules/poll_box.php:92]: The core will now handle template set IDs so this is causing problems.

**Comment by Tanweth** [comment]
My apologies for the lack of activity on this. GitHub has not been notifying me when new issues have been posted, so I didn't realize there was any activity on here. I'm about to post a major update to the module, and I'll be sure to include the suggested edits. Thank you for helping to make this a better plugin as always. :)

**Comment by WildcardSearch** [comment]
No probs.

Just a heads up @Tanweth I am considering some minor changes to the way modules work. Should I act on those changes (and I feel that I will), I'll submit a pull request with the changes you'll need to make. You obviously shouldn't feel pressured to pull it, I just want to keep you up to date.

Just FYI, I am looking to lighten the amount of data needed per module by foregoing passing the entire setting array to the modules and instead sending them only an array of setting values. So `$setting['setting_name']['value']` becomes `$setting['setting_name']` which is more intuitive anyway. :smile:

**Comment by Tanweth** [comment]
I'd be happy with that improvement. Thanks for the heads up.

**Pull request closed without merging**